### ISSUE TITLE
error syncing with EWS

### ISSUE THREAD

**Issue by mmiat** [ankeshdave/CalendarSyncplus#36]
Hi
I've this error when I try to sync with my office 365 account
![screenshot 2015-11-30 23 26 03](https://cloud.githubusercontent.com/assets/11910225/11486558/2e2f1130-97ba-11e5-9d5d-5cd431a3c5bf.png)

**Comment by mmiat** [comment]
is this project still alive?

**Comment by akankshagaur** [comment]
Yes, this project is alive. EWS is still under development and will be supported in future releases. Development is currently going on for existing bug fixes. Additional features will be added after that.

**Comment by mmiat** [comment]
ok thanksif you need some info (logs, test) let me know ;)




----Messaggio originale----

Da: notifications@github.com

Data: 04/12/2015 18.46

A: "ankeshdave/CalendarSyncplus"<CalendarSyncplus@noreply.github.com>

Cc: "mmiat"<mmiat@libero.it>

Ogg: Re: [CalendarSyncplus] error syncing with EWS (#36)




Yes, this project is alive. EWS is still under development and will be supported in future releases. Development is currently going on for existing bug fixes. Additional features will be added after that.

**Comment by mmiat** [comment]
but EWS now doesn't work at all, or maybe my configuration is wrong? thanks
